sdk/nodejs/__tests__/cloudClient.spec.ts

```diff
@@ -251,9 +254,10 @@ describe('DVCCloudClient with EdgeDB Enabled', () => {
             expect(res).toEqual({})
         })
 
-        it('to return no features when allFeatures call fails', async () => {
```

Comment:
seems like this test should still exist? its testing a different case than the invalid user one

Reply:
the tests time out for 500 level errors because it retries and we don't have an option to turn off retries from the client level, so now since the 400s throw we don't have a good way to test this (other than having the test timeout set to 10+ seconds)